src/Components/InputForm.js

```diff
@@ -27,6 +31,8 @@ const InputForm = () => {
       <input type="text" name="title" id="book-title" placeholder="title" required />
       <input type="text" name="author" id="book-author" placeholder="author" required />
       <input type="text" name="categories" id="book-categories" placeholder="categories" required />
+      <input type="text" name="progress" id="book-progress" placeholder="progress" required />
+      <input type="text" name="chapter" id="book-chapter" placeholder="chapter" required />
       <input type="submit" value="ADD BOOK" />
```

Comment:
[ Optional Changes ] I highly recommend you consider working on these changes. 

You did great by considering the other cases. I would say you went beyond the requirement and that's great 👍. It makes you learn and be good at things.

- [ ] It would be nice if you could use the `onChange` prop upon input fields value change. It was in the requirement though. Even though your implementation for adding books works fine, to follow instructions to details, you could have used the onChange prop on your input fields.
- [ ] Also, I would recommend you make set the type of the input field for book progress to a number, so one does not mistake to enter 89AHA as progress. 👍 You get the point 😂